distribution/src/assemble/component.xml

```diff
@@ -90,9 +90,6 @@
       <outputDirectory>jars</outputDirectory>
       <unpack>false</unpack>
       <useProjectArtifact>false</useProjectArtifact>
-      <includes>
-        <include>org.apache.drill:drill-shaded-guava:jar</include>
-      </includes>
     </dependencySet>
 
     <dependencySet>
```

Comment:
Please remove the whole `dependencySet` block, since it is not needed without `includes`.

Reply:
right, thanks!